sklearn/tree/tests/test_tree.py

```diff
@@ -1693,17 +1693,28 @@ def test_no_sparse_y_support(name):
 
 
 def test_mae():
-    # check MAE criterion produces correct results
+    # Check MAE criterion produces correct results
     # on small toy dataset
     dt_mae = DecisionTreeRegressor(random_state=0, criterion="mae",
                                    max_leaf_nodes=2)
+
+    # Test MAE where all sample weights are uniform:
+    dt_mae.fit([[3], [5], [3], [8], [5]], [6, 7, 3, 4, 3], np.ones(5))
```

Comment:
Cosmit: could you be explicit on the named kwargs, so that it is easy to see that you are setting sample weights.

Reply:
Yes I'll do that later, I have simply followed the style that pre-existed.